### DISCUSSION TITLE
Cypress: get dynamic ID

### DISCUSSION THREAD

**Discussion by MaxAllianov** [cypress-io/cypress#15322] [Questions and Help] [answered]
Hi everyone!

Please advise if there a way to get the elements with the dynamic part in the cypress?

I have to get the id like "Id=example-1q2w-example" where "1q2w" is a dynamic element that is updating every page reload.

It would be nice if this is possible to make a command like cy.get('#example-%%%%-example')

Cypress genius, looking for your help!

**Comment by tit** [comment, marked as answer]
Try 

```
// start with example- and end with -example
cy.get('[id^="example-"][id$="-example"]')
```

**Reply by MaxAllianov** [reply]
Thank you a lot!
I have modified it a little and it becomes work!

Here is a final working example:
 
`cy.get('[id^="example-"][id$="-example"]')`
